user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md

```diff
@@ -0,0 +1,284 @@
+---
+uid: DOM_Making_DOM_Stateful
+---
+
+# Making a DOM module state-aware
+
+This tutorial shows how you can make your DOM module state-aware. This allows a DOM instance to behave differently depending on the state it transitions to. The following things can be defined for each state:
+
+- Which fields are **visible** to the user.
+- Which fields are **read-only** for the user.
+- Which fields are **mandatory**.
+- To which state the instance can **transition**.
+- What **actions** are allowed.
+- What **buttons** to show on the form.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.3.
+
+Expected duration: 20 minutes
+
+> [!TIP]
+> See also: [Kata #15: Getting started with DataMiner Object Models](https://community.dataminer.services/courses/kata-15/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
```

Comment:
I noticed that this tutorial is for the upcoming Kata 24; will you update this link when the new Kata is available?